packages/admin-app/src/lang/MessageId.ts

```diff
@@ -27,25 +27,22 @@ enum MessageId {
     LoginFormSubmit = 'login_form.submit',
 
     // Election Setup
-    ElectionSetupManifestMenuTitle = 'election_setup.manifest_menu.title',
-    ElectionSetupManifestMenuAbout = 'election_setup.manifest_menu.about',
-    ElectionSetupManifestMenuPrompt = 'election_setup.manifest_menu.prompt',
-
     ElectionSetup_JointKeyUpload_Title = 'election_setup.joint_key_upload.title',
     ElectionSetup_JointKeyUpload_NoFile = 'election_setup.joint_key_upload.no_file',
     ElectionSetup_JointKeyUpload_InvalidFile = 'election_setup.joint_key_upload.invalid_file',
+    ElectionSetup_JointKeyUpload_SelectFiles = 'election_setup.joint_key_upload.select_files',
 
     ElectionSetup_JointKeySelect_Title = 'election_setup.joint_key_select.title',
     ElectionSetup_JointKeySelect_Prompt = 'election_setup.joint_key_select.prompt',
     ElectionSetup_JointKeySelect_Next = 'election_setup.joint_key_select.next',
 
+    ElectionSetupManifestPreviewId = 'election_setup.manifest_preview.property.id',
```

Comment:
Since you added this MessageId you might want to add the _ that you are putting into other MessageIds that you are changing.